content/en/guides/integrations/pytorch.md

```diff
@@ -40,7 +40,7 @@ for batch_idx, (data, target) in enumerate(train_loader):
         wandb.log({"loss": loss})
 ```
 
-If you need to track multiple models in the same script, you can call `wandb.watch` on each model separately. Reference documentation for this function is [here]({{< relref "/ref/python/watch.md" >}}).
+If you need to track multiple models in the same script, you can call `wandb.watch` on each model separately. See the [`wandb.watch` reference]({{< relref "/ref/python/watch.md" >}}) for more details.
```

Comment:
Might be a merge conflict with the other long PR:

```suggestion
If you need to track multiple models in the same script, you can call `wandb.Run.watch()` on each model separately. See the [`wandb.Run.watch()` reference]({{< relref "/ref/python/watch.md" >}}) for more details.
```

Reply:
Yeah your PR should go in first and I can deal with the conflicts in this PR.